hooks/post_gen_project.py

```diff
@@ -202,7 +202,8 @@ def update_local_workflows(central_workflows):
 
 
 def main():
-    add_supermodules(ROOT, "{{ cookiecutter.project_name }}")
+    if "." in "{{ cookiecutter.project_name }}":
+        add_supermodules(ROOT, "{{ cookiecutter.project_name }}")
     if "{{ cookiecutter.have_c_code }}" == "y":
```

Comment:
does this line need to be changed to the new name?

Reply:
I haven't changed the name:

<img width="341" alt="Screenshot 2024-12-28 at 5 12 25 PM" src="https://github.com/user-attachments/assets/df17cb6c-f542-4280-8ba8-2f6331df2702" />

I created an issue earlier to name `have_c_code` better, etc. in a separate PR

https://github.com/Billingegroup/cookiecutter/issues/213

Reply:
oops, sorry, I forgot that.